src/publishers/github.js

```diff
@@ -31,7 +31,8 @@ export default async (artifacts, packageJSON, forgeConfig, authToken, tag) => {
           repo: forgeConfig.github_repository.name,
           tag_name: tag || `v${packageJSON.version}`,
           name: tag || `v${packageJSON.version}`,
-          draft: true,
+          draft: forgeConfig.github_repository.draft !== false,
```

Comment:
@stanlemon yeah, that's the problem that @MarshallOfSound is trying to solve in the CLI, because publish targets will have different settings, and defining CLI options can be fairly static.